src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -427,6 +460,7 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Null(model.GetSymbolInfo(def).Symbol);
             Assert.Null(model.GetDeclaredSymbol(def));
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
For comparison, for `null`, the `Type` is `null`, the `ConvertedType` is the inferred type and the constant is `null` (which agrees with `ConvertedType`):

```
        public void AssignNullToString()
        {
            string source = @"
class C
{
    static void Main()
    {
        string x = null;
        System.Console.Write(x);
    }
}
";
            var comp = CreateCompilationWithMscorlib(source, parseOptions: TestOptions.ExperimentalParseOptions);
            comp.VerifyDiagnostics();

            var tree = comp.SyntaxTrees.First();
            var model = comp.GetSemanticModel(tree);
            var def = tree.GetCompilationUnitRoot().DescendantNodes().OfType<LiteralExpressionSyntax>().Single();

            Assert.Null(model.GetConstantValue(def).Value);
            Assert.Equal("System.String", model.GetTypeInfo(def).ConvertedType.ToTestDisplayString());
            Assert.Null(model.GetTypeInfo(def).Type);
        }
```